Dockerfile

```diff
@@ -4,69 +4,47 @@
 # https://github.com/crate/docker-crate
 #
 
-FROM alpine:3.7
+FROM centos:7
```

Comment:
Personally I'd go with a very explizit version of the base image to get reliable results.

Reply:
How do you mean? As in something like the `7.5.1804` tag, rather than just `7`?

Reply:
Yes, exactly.